src/blocks/card.css

```diff
@@ -0,0 +1,75 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
It would be great to use variables for all main values, such as main sizes, colors etc.